.github/workflows/docker.yml

```diff
@@ -47,14 +47,21 @@ jobs:
           echo "EVENT NAME: ${{ github.event.name }}"
           echo "MERGED: ${{ github.event.pull_request.merged }}"
 
+      # create the docker images with the user and group IDs
+      # set to 55172 (randomly chosen number unlikely to conflict
+      # with existing users). this helps to set permissions local
+      # file management with pulled images. Specifically, the
+      # following will make the xfer directory writeable by the
+      # images:
+      #   sudo chown -R 55172:55172 $PDO_SOURCE_ROOT/docker/xfer
```

Comment:
hmm, isn't just changing group the only thing you need?  _But_ you also would have to make sure it is group _writeable_ and the actual user running it is in that new group?  (i.e., you might also want to add the convenience comments on the sudo groupadd/usermod?)

Also, why not even add a make command to do these changes? And i guess there should ideally also be a docu-update?